### ISSUE TITLE
Register the achlys-ubuntu image on docker hub.docker.com 

### ISSUE THREAD

**Issue by FreddyWordingham** [digiLab-ai/achlys-uq#1]
TL;DR: We need `achlys-ubuntu` to be on a Docker registry so that we can build an image derived `FROM achlys-ubuntu`

So it turns out that by design Docker doesn't allow you to build a parent image, and then use that image as a base image for another container in your `docker-compose.yml` file. Ostensibly this is because parent images shouldn't change very often, but I wonder if it's a way of Docker pushing you to upload your container into their registry

We can get around this by either doing something similar to what @mikkelbue already has done with a shim installation, or more correctly you can create and run a local "registry" container, build and publish the `achlys-ubuntu` image to it, and then build our `achlys-uq` image (which is derived from `FROM achlys-ubuntu`)

Alternatively if `achlys-ubuntu` was publically avalible on [hub.docker.com](https://hub.docker.com/) then we wouldn't need to do any of this. But ofcouse there might be reasons why the authors of `achlys` have chosen not to do this

We couldn't do it ourselves as they have chosen to use a GPL license

**Comment by FreddyWordingham** [comment]
Don't think we need to do this anymore - we can host our own registry If need be
